alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/event/base/BaseEvent.java

```diff
@@ -92,4 +98,8 @@ public String toString() {
         ", createTime=" + createTime +
         '}';
   }
+
+  public Map<String,Object> toMap() throws Exception {
+    return mapper.readValue(mapper.writeValueAsString(this), Map.class);
```

Comment:
I think we should set the Date object serialization format, currently it's output is epoch millis like this: `createTime=1564213244165`. Not sure it's what you want.

Reply:
Don't worry, long type dates are good for data exchange.